### DISCUSSION TITLE
Comment simuler l'intégration des membres dans un groupe télégram par un bot telegram

### DISCUSSION THREAD

**Discussion by Jarodak47** [python-telegram-bot/python-telegram-bot#2790] [Q&A]
création d'un bot télégram qui simule l'intégration des membres dans un groupe télégram

**Comment by Bibo-Joshi** [comment]
Hi. Since this is an international platform, where most communication happens in english, I'd like to ask you to formulate your question in english as well. Additionally, I'd like to point you to [this wiki page](../wiki/Ask-Right) on how to ask well-posed question.
